packages/python/readme_metrics/Metrics.py

```diff
@@ -2,36 +2,42 @@
 import threading
 import requests
 import json
-from pprint import pprint
 
 from werkzeug import Request
 from pip._vendor.requests import __version__
 
-from metrics import MetricsApiConfig, ResponseInfoWrapper
-from metrics.PayloadBuilder import PayloadBuilder
+from readme_metrics import MetricsApiConfig, ResponseInfoWrapper
+from readme_metrics.PayloadBuilder import PayloadBuilder
 
 
 class Metrics:
     """
-    This is the internal central controller class invoked by the WSGI middleware. It handles the creation, queueing,
+    This is the internal central controller classinvoked by the WSGI
+    middleware. It handles the creation, queueing,
     and submission of the requests.
     """
     PACKAGE_NAME: str = 'readme/metrics'
     METRICS_API: str = 'https://metrics.readme.io'
 
     def __init__(self, config: MetricsApiConfig):
         """
-        Constructs and initializes the ReadMe Metrics controller class with the specified configuration.
+        Constructs and initializes the ReadMe Metrics controller class with
+        the specified configuration.
         :param config: Running configuration
         """
 
         self.config = config
-        self.payload_builder = PayloadBuilder(config.BLACKLIST, config.WHITELIST, config.IS_DEVELOPMENT_MODE, config.GROUPING_FUNCTION)
+        self.payload_builder = PayloadBuilder(
+                                config.BLACKLIST,
```

Comment:
These for PEP8?

Reply:
It's good according to flake8. I didn't resolve all of the linting problems though...

Reply:
k. I'm probably going to set up some lint checks in a workflow for the unit tests, along with [Pyre](https://pyre-check.org/) and [Pysa](https://pyre-check.org/docs/pysa-basics)